docs/quickstart.rst

```diff
@@ -0,0 +1,16 @@
+ABC-Classroom Quickstart
+------------------------
+
+To set up a template directory to run abc-classroom in, run:
+
+    $ abc-quickstart
+
+This will create a directory for you that contains pre-made directories for templates and cloned files, as well as
+a sample ``config.yml`` file that can be modified to run the program for your classroom. ``abc-quickstart`` has
```

Comment:
i think it should create the needed config file not a sample, the user then has to followup by customizing that file.